### DISCUSSION TITLE
Installin Nitrux from bootable USB gets stuck at User's screen

### DISCUSSION THREAD

**Discussion by GraceLGriff** [Nitrux/nitrux-bug-tracker#171] [Q&A] [answered]
# Issue Description
When installing Nitrux da08b65b-amd64 from a USB stick created from Linux Mint's USB Image Writer app, the process gets to the User's details screen, and no matter is entered, the NEXT button will not become active.

**Describe the bug**

Please make sure to include a clear and concise description of what the bug is. 
When installing Nitrux da08b65b-amd64 from a USB stick created from Linux Mint's USB Image Writer app on a Lenovo T430, the process gets to the User's details screen, and no matter is entered, the NEXT button will not become active.

**Steps to reproduce the bug**

1. Create bootable USB stick on Lenovo T430, running Linux Mint 22, using the build in USB Image Writer
2. Boot Lenovo T430 from the above bootable USB stick.
3. Make cursor visible by manipulating red "nib".
4. Connect to internet.
5. Click on the Install Nitrux icon.
6. Click Next.
7. Select Region/Timezone.
8. Next.
9. Select keyboard "Generic PC 86 key".
10. Next.
11. Select keyboard layout.
12. Next.
13. Enter full name - Grace
14. Enter computer name - Lenovo-T430.
15. Enter user password twice - TestPa55word
16. Enter root password twice - TestPa55word.
17. NEXT key is not available.

**Expected behavior**

Could you describe what you expected to happen when the bug occurred?
Move onto the next stage of the installation.

**Screenshots**

Could you include screenshots to help explain your problem?
Was not possible to save screen shot onto the USB stick or access the computer's home directory.
It's all pretty explanatory though. The NEXT button does not become active.

**Comment by UriHerrera** [comment, marked as answer]
Please check the [installation tutorial on the website](https://nxos.org/tutorial/how-to-install-nitrux/).

- Use an adequate password that passes the password validation or uncheck the password validation in the Calamares installer.
- The available target storage may use an MBR partition table instead of GPT. MBR limits the available partitions to four primary partitions; thus, we discourage using an MBR partition table. You can use a different target device or create a new partition table on the target device.

Please also include the Calamares log the installer generates in the Live session. The path to the file is listed in the issue template and the tutorial.

----

I'm transferring this to a discussion because it's not a bug.
